src/main/java/mServer/crawler/sender/wdr/WdrSendungOverviewDeserializer.java

```diff
@@ -0,0 +1,91 @@
+package mServer.crawler.sender.wdr;
+
+import mServer.crawler.CrawlerTool;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class WdrSendungOverviewDeserializer extends WdrDeserializerBase {
+
+    private static final String QUERY_URL = "div.hideTeasertext > a";
+    private static final String QUERY_URL_LOKALZEIT_VIDEO_TEASER = "div.teaser.video > a";    
+    private static final String QUERY_URL_LOKALZEIT_MEHR = "h3.headline > a";
+    private static final String QUERY_URL_ROCKPALAST_YEARS = "div.entries > div";
+    private static final String QUERY_URL_ROCKPALAST_YEARS_ENTRIES = "div.entry > a";
+    private static final String QUERY_URL_TYPE = "p.teasertext > strong";
+    
+    public WdrSendungDto deserialize(Document document) {
+        WdrSendungDto dto = new WdrSendungDto();
+
+        addUrls(dto, document, QUERY_URL, UrlType.VideoPage);
+        addUrls(dto, document, QUERY_URL_LOKALZEIT_VIDEO_TEASER, UrlType.VideoPage);
+        addUrls(dto, document, QUERY_URL_LOKALZEIT_MEHR, UrlType.VideoPage);
+        
+        if(checkParseRockpalastYears(document)) {
+            addUrls(dto, document, QUERY_URL_ROCKPALAST_YEARS_ENTRIES, UrlType.OverviewPage);
+        }
+            
+        return dto;
+    }
+    
+    private void addUrls(WdrSendungDto dto, Document document, String query, UrlType defaultUrlType) {
+        Elements urlElements = document.select(query);
+        urlElements.forEach(urlElement -> {
+            String url = urlElement.attr(HtmlDeserializerBase.HTML_ATTRIBUTE_HREF);
+            if(url != null && !url.isEmpty()) {
+                url = addDomainIfNecessary(url);
+                
+                switch(getUrlType(urlElement, defaultUrlType)) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)